tests/unit/serve/executors/test_bad_executor_constructor.py

```diff
@@ -13,7 +13,7 @@ def foo(self, **kwargs):
 
 
 class GoodExecutor2(Executor):
-    def __init__(self, metas, requests, runtime_args):
+    def __init__(self, metas, requests, runtime_args, dynamic_batching):
```

Comment:
what are these `dynamic_batching`? why not part of `runtime_args`? @alaeddine-13 @JohannesMessner